src/components/DetailsBanner.jsx

```diff
@@ -0,0 +1,15 @@
+export default function DetailsBanner({ name, image }) {
+  const { src, alt } = image; // destructure from image object
+  return (
+    <div className="details-banner">
+      {name}
```

Comment:
Axe Devtools is asking for a level one header on the page. I'd wrap the {name} here in an h1 tag to see if that solves the issue.